<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchDetectorDialog</class>
 <widget class="QDialog" name="BatchDetectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量添加探测器</string>
  </property>
  <widget class="QPushButton" name="ptnDelete">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>111</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnModify">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>111</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnAdd">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>111</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnSave">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>640</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnClose">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="detectorTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>620</width>
     <height>470</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>探测器编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>探测器信道编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>阀值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>延长时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>是否重启探测器</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>631</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>探测器编号*：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtDetectorNo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>由0-9的数字组成，固定长3位</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>探测器信道编号*：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtDetectorChannelID"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string>由0-9的数字组成</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>阀    值：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbxThreshold">
          <property name="maximum">
           <number>65536</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>延长时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbxDelayTime">
          <property name="maximum">
           <number>65536</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>是否重启探测器：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbReset">
          <item>
           <property name="text">
            <string>是</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>否</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>备注：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtDetectorComments"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>ptnDelete</zorder>
  <zorder>ptnModify</zorder>
  <zorder>ptnAdd</zorder>
  <zorder>ptnSave</zorder>
  <zorder>ptnClose</zorder>
  <zorder>detectorTableWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
